dcpy/lifecycle/package/generate_metadata_assets.py

```diff
@@ -1,54 +1,64 @@
-from dcpy.models.product.dataset.metadata import Metadata, SocrataMetada
-import argparse
 from pathlib import Path
-from jinja2 import Environment, FileSystemLoader
+from jinja2 import Template
 import typer
 import subprocess
+from dcpy.models.product.dataset.metadata import Metadata
+
+DEFAULT_DATA_DICTIONARY_TEMPLATE_PATH = (
+    Path(__file__).parent / "resources" / "data_dictionary_template.jinja"
+)
 
 app = typer.Typer()
 
 
-@app.command()
-def generate_pdf_from_yml(
+@app.command("pdf_from_yml")
+def _cli_wrapper_pdf_from_yml(
     yaml_file_path: str,
     html_template_path: str,
     output_html_path: str,
     output_pdf_path: str,
-    pdf_metadata_path,
-) -> Path:
-    generate_html_from_yaml(yaml_file_path, html_template_path, output_html_path)
+) -> None:
+    generate_html_from_yaml(
+        Path(yaml_file_path),
```

Comment:
Any reason not to make these `Path` objects in the function args? Typer will take care of handling converting them.

Reply:
~~that sounds good, I'll change the args. but I didn't wanna worry about optimizing the CLI stuff here yet~~

per the use in `template/build_scripts.export.py` of the non-CLI functions, a datasets's metadata has to be downloaded to be converted to a PDF. so these CLI functions aren't very usable in a build

I just deleted the CLI part of this so that we can invoke this sub-step of packaging with higher-level CLI command later